cmd/podman-wslkerninst/main.go

```diff
@@ -85,7 +88,10 @@ func warn(title string, caption string) int {
 
 func main() {
 	args := os.Args
-	setupLogging(path.Base(args[0]))
+	if _, err := setupLogging(path.Base(args[0])); err != nil {
+		fmt.Fprintf(os.Stderr, "Error configuring logging: %v", err)
+		// Don't exit, it's just a logging issue
```

Comment:
Not sure if we want to actually log this, https://github.com/containers/podman/issues/21426 seems to suggest that there are people running without event logger. cc @n1hility

Reply:
yeah where this is ran its not actually visible (no console attached when ran from the installer), so it would only be visible for manual execution. If it were me I would just toss the error.

Re the issue. It's pretty atypical for Windows Logging to be not running since its an essential service (granted we shouldn't fail if its not running for exceptional reasons)